openmetadata-airflow-apis/openmetadata_managed_apis/api/routes/delete.py

```diff
@@ -37,13 +38,15 @@ def delete_dag() -> Response:
     }
     """
     dag_id = get_arg_dag_id()
+    # Sanitize Path Traversal, as later on we clean files based on the DAG id
+    secure_dag_id = secure_filename(dag_id)
```

Comment:
Interesting, what does it do exactly? `secire_filename`?

Reply:
makes sure that people don't go from file to file, so something like `../../../system32` would be just `system32`